src/cryptography/hazmat/backends/openssl/decode_asn1.py

```diff
@@ -710,6 +710,10 @@ def _asn1_string_to_utf8(backend, asn1_string):
     buf = backend._ffi.new("unsigned char **")
     res = backend._lib.ASN1_STRING_to_UTF8(buf, asn1_string)
     if res == -1:
+        str_bytes = _asn1_string_to_bytes(backend, asn1_string)
+        if asn1_string.type == int('0x12', 16):
```

Comment:
`ASN1_STRING_to_UTF8` uses `ASN1_mbstring_copy` which according to
[0e2d0911](https://git.openssl.org/?p=openssl.git;a=commit;h=0e2d091103ed5846c72a5c9e8b10e25ead4bf152) should convert NumericString, but `tag2nbyte` in [crypto/asn1/a_strex.c](https://git.openssl.org/?p=openssl.git;a=blob_plain;f=crypto/asn1/a_strex.c;hb=0e2d091103ed5846c72a5c9e8b10e25ead4bf152) was not updated to pass NumericString to `ASN1_mbstring_copy` yet and I think it should be updated.
So this patch is workaround.